EventFlow.ReadStores.ElasticSearch.Tests/IntegrationsTests/EsIntegrationTestConfiguration.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+using EventFlow.Configuration;
+using EventFlow.EventStores.MsSql;
+using EventFlow.MsSql;
+using EventFlow.MsSql.Extensions;
+using EventFlow.MsSql.Tests.Helpers;
+using EventFlow.ReadStores.ElasticSearch.Extensions;
+using EventFlow.Test;
+using EventFlow.Test.Aggregates.Test;
+using EventFlow.Test.Aggregates.Test.ReadModels;
+using Nest;
+using TestAggregateReadModel = EventFlow.ReadStores.ElasticSearch.Tests.ReadModels.TestAggregateReadModel;
+
+namespace EventFlow.ReadStores.ElasticSearch.Tests.IntegrationsTests
+{
+    public class EsIntegrationTestConfiguration : IntegrationTestConfiguration
+    {
+        protected ITestDatabase TestDatabase { get; private set; }
+        protected IMsSqlConnection MsSqlConnection { get; private set; }
+        protected IElasticClient ElasticClient { get; private set; }
+        
+        public override IRootResolver CreateRootResolver(EventFlowOptions eventFlowOptions)
+        {
+            TestDatabase = MsSqlHelper.CreateDatabase("eventflow");
+
+            var resolver = eventFlowOptions
+                .ConfigureMsSql(MsSqlConfiguration.New.SetConnectionString(TestDatabase.ConnectionString))
+                .ConfigureElasticSearch(EsConfiguration.New.SetConnectionString("http://127.0.0.1:9200"))
```

Comment:
@tabro Check to see if the environment variable `ELASTICSEARCH_URL` has a value and if it does, use that. If not, then use localhost.